healthcare/api-client/v1/consent/README.rst.in

```diff
@@ -10,7 +10,9 @@ product:
     data with cloud-based applications.
 
 required_api_url: https://console.cloud.google.com/apis/library/healthcare.googleapis.com
-required_role: Healthcare Consent Store Administrator
```

Comment:
Hmm... I wonder if the readme generator broke? Doesn't seem like you did anything particularly different to break so much of the readme...